util/mdbook_reggen.py

```diff
@@ -31,13 +31,13 @@ def main() -> None:
     book_root = context["root"]
 
     try:
-        ip_cfg_str = context["config"]["preprocessor"]["reggen"]["ip-cfg-py-regex"]
+        ip_cfg_str = context["config"]["preprocessor"]["reggen"][
```

Comment:
Did the auto-formatter break this line? Odd choice :eyes:

Reply:
yes, it is rater nasty isn't it.